bin/upgrade

```diff
@@ -160,8 +160,25 @@ function handle_image_upgrade() {
   set_server_pro_image_name "$SEED_IMAGE_VERSION"
   docker pull "$IMAGE"
   if [[ $GIT_BRIDGE_ENABLED == "true" ]]; then
-    set_git_bridge_image_name "$SEED_IMAGE_VERSION"
-    docker pull "$GIT_BRIDGE_IMAGE"
+    if [[ -n ${GIT_BRIDGE_IMAGE:-} ]]; then
+      echo "-------------------  WARNING  ----------------------"
+      echo "  You're using a custom git bridge image: $GIT_BRIDGE_IMAGE"
+      echo "  Upgrade of your custom image is skipped. You need to pull the image separately, making sure that:"
+      echo "     1. The Docker image is tagged with the new version: $SEED_IMAGE_VERSION"
+      echo "     2. The environment variable GIT_BRIDGE_IMAGE only contains the image name, and not a tag/version."
+      echo "  You can safely proceed with the upgrade process, but if you try subsequently to restart your instance"
+      echo "  it will fail until the updated image with tag $SEED_IMAGE_VERSION is available in your system"
```

Comment:
Is it "safe" when the upgrade fails? :) I think we can omit this part.

Reply:
Well, it's safe provided you edit manually `GIT_BRIDGE_IMAGE` as requested in the prompt, but reading again the full context I agree it's better to omit.

```suggestion
      echo "  If you try subsequently to restart your instance it will fail until the updated image with"
      echo "  tag $SEED_IMAGE_VERSION is available in your system"
```